test/index.js

```diff
@@ -61,18 +61,18 @@ const DUMMY_PACKAGE_JSON = { name: 'dummy', version: '1.0.0' }
 const netlifyConfig = { build: {} }
 
 describe('preBuild()', () => {
-  test('fail build if the app has static html export in npm script', async () => {
-    await expect(
-      plugin.onPreBuild({
-        netlifyConfig: { build: { command: 'npm run build' } },
-        packageJson: { ...DUMMY_PACKAGE_JSON, scripts: { build: 'next export' } },
-        utils,
-        constants: { FUNCTIONS_SRC: 'out_functions' },
-      }),
-    ).rejects.toThrow(
-      `Static HTML export Next.js projects do not require this plugin. Check your project's build command for 'next export'.`,
-    )
-  })
+  // test('fail build if the app has static html export in npm script', async () => {
+  //   await expect(
+  //     plugin.onPreBuild({
+  //       netlifyConfig: { build: { command: 'npm run build' } },
+  //       packageJson: { ...DUMMY_PACKAGE_JSON, scripts: { build: 'next export' } },
+  //       utils,
+  //       constants: { FUNCTIONS_SRC: 'out_functions' },
+  //     }),
+  //   ).rejects.toThrow(
+  //     `Static HTML export Next.js projects do not require this plugin. Check your project's build command for 'next export'.`,
+  //   )
+  // })
```

Comment:
Should this be removed instead of being commented out?
We can always retrieve the old code by using `git checkout` or going back to this GitHub PR.

Reply:
i left them commented per my question/thoughts in the PR body re: these tests :)